tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -0,0 +1,516 @@
+PRIVATE_REGISTRY_USERNAME = os.environ.get("RANCHER_BASTION_USERNAME")
+PRIVATE_REGISTRY_PASSWORD = \
+    os.environ.get("RANCHER_BASTION_PASSWORD", ADMIN_PASSWORD)
+RANCHER_EXTERNAL_BASTION = os.environ.get("RANCHER_EXTERNAL_BASTION", "")
+RANCHER_EXTERNAL_HOST_NAME = os.environ.get(
+    "RANCHER_EXTERNAL_HOST_NAME", AG_HOST_NAME)
+REG_HOST_NAME = RANCHER_EXTERNAL_HOST_NAME + "-registry"
+REGISTRY_HOSTNAME = os.environ.get(
+    "REGISTRY_HOSTNAME", REG_HOST_NAME + ".qa.rancher.space")
+
+
+def test_deploy_airgap_rancher(check_hostname_length):
+    # check for user entered bastion/registry
+    if len(RANCHER_EXTERNAL_BASTION) > 5:
+        bastion_node = AmazonWebServices().get_node(
+            RANCHER_EXTERNAL_BASTION,
+            ssh_access=True)
+        print("using external node for private registry")
+    else:
+        bastion_node = deploy_bastion_server()
+        add_rancher_images_to_private_registry(
+                                        bastion_node,
+                                        noauth_reg_name=REGISTRY_HOSTNAME)
+        install_rke_helm_kubectl(bastion_node)
+        print("using new node for private registry")
+
+    # check for registry, kubectl, helm, images, rke
+    assert len(bastion_node.private_ip_address) > 5, \
+        "the bastion node does not have a private IP"
+    assert bastion_node.execute_command("docker pull {}/{}:{}".format(
+        REGISTRY_HOSTNAME,
+        "rancher/rancher-agent",
+        RANCHER_SERVER_VERSION))[1].find("not found") < 0, \
+        "registry is missing rancher-agent image"
+    assert bastion_node.execute_command(
+        "rke --version")[1].find("not found") < 0, \
+        "registry does not have RKE installed"
+    assert bastion_node.execute_command(
+        "/snap/bin/helm version")[1].find("not found") < 0, \
+        "registry does not have helm installed"
+    kube_config = 'export KUBECONFIG="kube_config_config.yaml && '
+    assert bastion_node.execute_command(
+        kube_config+"/snap/bin/kubectl version")[1].find("not found") < 0, \
+        "registry does not have kubectl installed"
+
+    ag_nodes = setup_airgap_rancher(bastion_node)
+    public_dns = create_nlb_and_add_targets(ag_nodes)
+    print(
+        "\nConnect to bastion node with:\nssh -i {}.pem {}@{}\n"
+        "Connect to rancher node by connecting to bastion, then run:\n"
+        "ssh -i {}.pem {}@{}\n\nOpen the Rancher UI with: https://{}\n"
+        "** IMPORTANT: SET THE RANCHER SERVER URL UPON INITIAL LOGIN TO: {} **"
+        "\nWhen creating a cluster, enable private registry with below"
+        " settings:\nPrivate Registry URL: {}\nPrivate Registry is not"
+        "using basic auth (no login required) \n".format(
+            bastion_node.ssh_key_name, AWS_USER, bastion_node.host_name,
+            bastion_node.ssh_key_name, AWS_USER,
+            ag_nodes[0].private_ip_address,
+            public_dns, RANCHER_AG_INTERNAL_HOSTNAME,
+            bastion_node.host_name))
```

Comment:
A quick change here to use the `RANCHER_AG_HOSTNAME`  instead of the `bastion_node.host_name` as the private registry we should be using the Route53 record while creating a Custom Cluster.

No need to re-run the automation job for this after the change though.

Reply:
good catch, I've updated that in the latest version